app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -19,25 +14,61 @@
   &:focus {
     background-color: $color-white;
   }
+}
 
-  @include respond-to($mq-xs) {
-    display: none;
-  }
-
+.chat-box__container {
+  top:30%;
+  left: 0;
+  padding: $baseline-unit*2;
+  border: 2px solid $color-green-primary;
+  border-radius: 0 4px 4px 0;
+  border-left-style: none;
+  display: none;
+  // min-width 960
   @include respond-to($mq-l) {
     display: block;
   }
-}
 
-.chat-box__heading {
-  color: $color-heading-extra-small;
-  font-weight: 500;
-  font-size: 1.2rem;
-  margin-bottom: 5px;
+  .chat-box__heading {
+    color: $color-heading-extra-small;
+    font-weight: 500;
+    font-size: 1.2rem;
+    margin-bottom: 5px;
+  }
+  
+  .chat-box__icon {
+    margin-left: -35px;	
+  }
 }
 
-.chat-box__icon {
-  margin-left: -35px;	
+.chat-popup__container {
+  bottom: 2%;
+  right: 2%;
```

Comment:
Can we use the $default-gutter variable for this value?

Reply:
Corrected